minifi/minifi-docs/src/main/markdown/System_Admin_Guide.md

```diff
@@ -899,7 +897,7 @@ Security Properties:
     ssl protocol: TLS
     Sensitive Props:
         key:
-        algorithm: PBEWITHMD5AND256BITAES-CBC-OPENSSL
+        algorithm: NIFI_PBKDF2_AES_GCM_256
         provider: BC
```

Comment:
should the provider be removed?

Reply:
Thanks for catching that, yes, it should be removed.